salt/lax/bot-lax-adaptor.sls

```diff
@@ -149,22 +150,30 @@ bot-lax-uwsgi-conf:
             - bot-lax-adaptor-install
             - bot-lax-writable-dirs
 
-{% set apiprotocol = 'https' if salt['elife.cfg']('cfn.outputs.DomainName') else 'http' %}
-{% set apihost = salt['elife.cfg']('project.full_hostname', 'localhost') %}
-
-uwsgi-bot-lax-adaptor:
+bot-lax-uwsgi-upstart:
     file.managed:
         - name: /etc/init/uwsgi-bot-lax-adaptor.conf
         - source: salt://lax/config/etc-init-uwsgi-bot-lax-adaptor.conf
         - template: jinja
         - mode: 755
 
+bot-lax-uwsgi-systemd:
+    file.managed:
+        - name: /lib/systemd/system/uwsgi-bot-lax-adaptor.service
+        - source: salt://lax/config/lib-systemd-system-uwsgi-bot-lax-adaptor.service
+        - template: jinja
+        - mode: 644
```

Comment:
systemd supports templating services with an example here: https://serverfault.com/questions/730239/start-n-processes-with-one-systemd-service-file

more here: https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers

I'll get to that when it starts failing (hopefully) on an 16.04 aws instance